ui/components/src/LineChart/LineChart.tsx

```diff
@@ -150,6 +163,17 @@ export function LineChart({ height, data, unit, grid, legend, visualMap, onDataZ
 
     const rangeMs = data.rangeMs ?? getDateRange(data.xAxis);
 
+    const yAxisDefault = {
```

Comment:
Nit: I think we've mostly been using SCREAMING_SNAKE for constants, so this might be more consistent as `Y_AXIS_DEFAULT` or something like that.

Also, since this doesn't depend on any of the props, could it get pulled outside of the function to avoid being redefined on every render cycle?